tck/src/main/java/org/eclipse/microprofile/lra/tck/LraTckConfigBean.java

```diff
@@ -34,6 +34,45 @@
      */
     private static final Long LRA_TIMEOUT_MILLIS = 50000L;
 
+    /**
+     * Name of the system property that is used as factor adjusting timeout values
+     * used in the testsuite.
+     */
+    private static final String LRA_TCK_TIMEOUT_FACTOR_PROPETY_NAME = "lra.tck.timeout.factor";
```

Comment:
I wonder whether we should prefix every (even tck) property with `mp.lra.` to be aligned with general MP config names. I will do that in different PR.